ReactAndroid/src/main/java/com/facebook/react/views/text/ReactInlineBorderSpan.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.views.text;
+
+import android.graphics.Paint;
+import android.text.style.LineBackgroundSpan;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.graphics.Color;
+
+/**
+ * We use a custom {@link LineBackgroundSpan}, because RN Text component can't render borders. Details here:
+ * https://github.com/Expensify/App/issues/4733
+ */
+public class ReactInlineBorderSpan implements LineBackgroundSpan, ReactSpan {
+  private final int effectiveStart;
+  private final int effectiveEnd;
+  private final int effectiveFontSize;
+
+  private final int backgroundColor = Color.parseColor("#f6f6f6");
+  private final int borderColor = Color.parseColor("#b3b8bc");
```

Comment:
We should update these now that we default to the dark theme. Please pull main and update these. Also, adding a comment might be useful

Reply:
Yeap, gonna allow passing those from JS.

Reply:
Done. It supports this format from now on:
```js
<Text
  style={{color: '#fff'}}
  textCodeBlock={{
    backgroundColor: '#333333',
    borderColor: '#FFFF00',
    borderRadius: 8,
  }}>
  This text should wrapped with a border and displayed inline.
</Text>
```